sympy/logic/boolalg.py

```diff
@@ -1368,6 +1475,59 @@ def to_dnf(expr, simplify=False):
     return distribute_or_over_and(expr)
 
 
+def is_anf(expr):
+    """
+    Checks if expr is in Algebraic Normal Form (ANF).
+
+    A logical expression is in ANF if it has the form
+    ((A & B & ...) ^ (B & C ...) ^ B ^ True ^ ...),
+    i.e. it is purely true, purely false, conjunction of
```

Comment:
Can `expr` be only one of them or it can be more than one? It isn't clear to me.

Reply:
`expr` can be:
- purely true
- purely false
- conjunction of variables: A & B & C & D ...
- exclusive disjunction: ((A & B & ...) ^ (B & C ...) ^ B ^ True ^ ...). The exclusive disjunction can only contain true, variables or a conjunction of variables.

Reply:
I think putting them in the next line, as you have done in comment, in this way would be better.

Reply:
Done.